src/sinks/mod.rs

```diff
@@ -35,6 +35,8 @@ pub mod elasticsearch;
 pub mod file;
 #[cfg(feature = "sinks-gcp")]
 pub mod gcp;
+#[cfg(any(feature = "sinks-gcp", feature = "sinks-datadog_archives"))]
```

Comment:
You can simply have it look for `sinks-gcp` since you include that feature flag via `sinks-datadog_archives`.

Reply:
indeed, updated